repose-aggregator/artifacts/web-application/src/main/java/org/openrepose/core/ReposeInitializer.java

```diff
@@ -91,5 +91,9 @@ public void onStartup(ServletContext servletContext) throws ServletException {
         servletContext.addServlet("emptyServlet", EmptyServlet.class).addMapping("/*");
         servletContext.addFilter("springDelegatingFilterProxy", new DelegatingFilterProxy("powerFilter"))
                 .addMappingForUrlPatterns(EnumSet.allOf(DispatcherType.class), false, "/*");
+
+        // todo: when switching to the new ReposeFilterChain, replce the previous line with the following line
+        // todo: to wire in the routing servlet
+        // servletContext.addServlet("reposeServlet", rootContext.getBean(ReposeServlet.class)).addMapping("/*");
```

Comment:
I think this comment is offset by a line. I'm still able to figure it out, but that was confusing for a second.